.github/workflows/performance.yml

```
@@ -0,0 +1,160 @@
+          echo "PHP_FPM_UID=$(id -u)" >> $GITHUB_ENV
+          echo "PHP_FPM_GID=$(id -g)" >> $GITHUB_ENV
+
+      - name: Checkout repository
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+
+      - name: Set up Node.js
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version-file: '.nvmrc'
+          cache: npm
+
+      - name: Log debug information
+        run: |
+          npm --version
+          node --version
+          curl --version
+          git --version
+          svn --version
+          locale -a
+
+      - name: Install npm Dependencies
+        run: npm ci
+
+      - name: Build WordPress
+        run: npm run build
+
+      - name: Start Docker environment
+        run: |
+          npm run env:start
+
+      - name: Log running Docker containers
+        run: docker ps -a
+
+      - name: Docker debug information
+        run: |
+          docker -v
+          docker-compose -v
+          docker-compose run --rm mysql mysql --version
+          docker-compose run --rm php php --version
+          docker-compose run --rm php php -m
+          docker-compose run --rm php php -i
+          docker-compose run --rm php locale -a
+
+      - name: Install WordPress
+        run: npm run env:install
+
+      - name: Install WordPress Importer plugin and import mock data
+        run: |
+          npm run env:cli -- plugin install wordpress-importer --activate
+          curl -O https://raw.githubusercontent.com/WPTT/theme-test-data/b9752e0533a5acbb876951a8cbb5bcc69a56474c/themeunittestdata.wordpress.xml
+          npm run env:cli -- import themeunittestdata.wordpress.xml --authors=create
+          rm themeunittestdata.wordpress.xml
+
+      - name: Update permalink structure
+        run: |
+          npm run env:cli -- rewrite structure '/%year%/%monthnum%/%postname%/'
+
+      - name: Install MU plugin
+        run: |
+          mkdir ./build/wp-content/mu-plugins
+          cp ./tests/performance/mu-plugins/server-timing.php ./build/wp-content/mu-plugins/server-timing.php
+
+      - name: Run Performance tests (current commit)
+        run: npm run test:performance
+
+      - name: Print Performance tests results
+        run: "node ./tests/performance/results.js"
+
+      - name: Update the environment to 6.1.1
+        run: |
+          npm run env:cli -- core update --version=6.1.1 --force --path=/var/www/build
+          npm run env:cli -- core version --path=/var/www/build
+
+      - name: Run Performance tests (6.1.1)
+        run: npm run test:performance -- --prefix=base
+
+      - name: Print Performance tests results (6.1.1)
+        run: "node ./tests/performance/results.js --prefix=base"
+
+      - name: Set commit details
+        uses: actions/github-script@98814c53be79b1d30f795b907e553d8679345975 # v6.4.0
+        id: commit-timestamp
+        with:
+            github-token: ${{secrets.GITHUB_TOKEN}}
```

```suggestion
            github-token: ${{ secrets.GITHUB_TOKEN }}
```

Super small nit to add spaces. Just matches the syntax used in the other expressions here and within other workflows.